sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -34,10 +34,10 @@ export interface ConnectionContextBase {
    */
   readonly negotiateClaimLock: string;
   /**
-   * @property {TokenProvider} tokenProvider The TokenProvider to be used for getting tokens
+   * @property {SharedKeyCredential | TokenCredential} tokenCredential The TokenCredential to be used for getting tokens
```

Comment:
Should the description be more generic since it accepts multiple types of credentials? Something like:
```suggestion
   * @property {SharedKeyCredential | TokenCredential} tokenCredential The credential to be used for getting tokens
```